etcdctl/README.md

```diff
@@ -1148,7 +1148,7 @@ Whenever a leader is elected, its proposal is given as output.
 
 ELECT returns a zero exit code only if it is terminated by a signal and can revoke its candidacy or leadership, if any.
 
-If a candidate is abnormally terminated, election rogress may be delayed by up to the default lease length of 60 seconds.
+If a candidate is abnormally terminated, election progress may be delayed by up to the default lease length of 60 seconds.
```

Comment:
Thanks for catching this @johnlolis To make build happy, can you please update your commit title i.e. change the current title with `doc:` prefix, for example,  `doc: Update etcdctl README` 
The build error is, fyi, `Expected commit title format <package>{", "<package>}: <description>`